SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -2047,39 +2047,6 @@ void CEulerSolver::Source_Residual(CGeometry *geometry, CSolver **solver_contain
     AD::EndNoSharedReading();
   }
 
-  if (rotating_frame) {
-
-    /*--- Include the residual contribution from GCL due to the static
-     mesh movement that is set for rotating frame. ---*/
-
-    SetRotatingFrame_GCL(geometry, config);
-
-    /*--- Loop over all points ---*/
-    AD::StartNoSharedReading();
-    SU2_OMP_FOR_DYN(omp_chunk_size)
-    for (iPoint = 0; iPoint < nPointDomain; iPoint++) {
-
-      /*--- Load the conservative variables ---*/
-      numerics->SetConservative(nodes->GetSolution(iPoint),
-                                nodes->GetSolution(iPoint));
```

Comment:
You are also taking out the rotating frame source term, not just the GCL.
Can you upload the entire configs for your cases?

Reply:
Tom Economon has a chapter on the GCL in his thesis https://github.com/economon/economon.github.io/tree/master/docs
I'll have a look at the configs and think about this a bit.